.github/workflows/docs.yml

```
@@ -0,0 +1,57 @@
+name: Deploy docs content to Pages 2
+
+on:
+  push:
+    branches: ["docs_build_v1"]
```

We probably want this to actually normally happen on pushes to main 

```suggestion
    branches: ["main", "docs_build_v1"]
```